--- cli.py	2016-11-16 11:25:05.173407000 -0800
+++ cli2.py	2016-11-16 11:27:12.885407000 -0800
@@ -167,10 +167,12 @@
         keep_trailing_newline=True,
     )
 
     # Add environ global
     env.globals['environ'] = os.environ.get
+    env.lstrip_blocks = True
+    env.trim_blocks = True
 
     output = env.get_template(os.path.basename(template_path)).render(data)
     return output.encode('utf-8')
 
 
